src/olympia/internal_tools/tests/test_views.py

```diff
@@ -45,6 +47,8 @@ def perform_search(
 
     def perform_search_with_senior_editor(
             self, url, data=None, expected_queries_count=3, **headers):
+        # Just so that the waffle switch db query is bypassed (it's cached).
```

Comment:
Maybe also note the `assertNumQuery` usage to make it easier to reason about it.